code/game/objects/effects/aliens.dm

```diff
@@ -135,6 +135,33 @@
 	acid_t = null
 	. = ..()
 
+/obj/effect/xenomorph/acid/attackby(var/obj/O as obj, var/mob/user as mob)
+	if(!istype(O, /obj/item/anti_acid))
+		..()
+		return
+
+	if(istype(O, /obj/effect/xenomorph/acid/strong))
```

Comment:
Is this check on if acid is attacked by acid?
Isn't this covered already in the previous check?